add-on/content/SerpMonitor.jsm

```diff
@@ -125,19 +131,20 @@ this.SerpMonitor = {
         return;
       }
 
-      Cu.reportError(`>>>> [${[...this.serpTabs.keys()]}]\n`);
-      triggerURI && Cu.reportError(`>>>> triggeringPrincipal in map: ${this.serpTabs.has(triggerURI.spec)}`)
-      triggerURI && Cu.reportError(`>>>> triggeringPrincipal: ${triggerURI.spec}`)
-      Cu.reportError(`>>>> channel.URI: ${uri.spec}`)
+      log.trace(`>>>> [${[...this.serpTabs.keys()]}]\n`);
+      if (triggerURI) {
+        log.trace(`>>>> triggeringPrincipal in map: ${this.serpTabs.has(triggerURI.spec)}`);
+        log.trace(`>>>> triggeringPrincipal: ${triggerURI.spec}`);
+      }
+      log.trace(`>>>> channel.URI: ${uri.spec}`);
       if (resultDomainInfo &&
         uri.filePath.substring(1).startsWith(resultDomainInfo.prefix) &&
                  this.serpTabs.has(triggerURI.spec)) {
         let info = this.serpTabs.get(triggerURI.spec);
-        Cu.reportError(`${resultDomainInfo.sap} ad click`); // XXX
 
         let histogram = Services.telemetry.getKeyedHistogramById("SEARCH_COUNTS");
+        log.info(`Reporting to Telemetry: ${info.sap}.adclick:unknown:${info.code}`);
         histogram.add(`${info.sap}.adclick:unknown:${info.code}`);
-        this.serpTabs.delete(triggerURI.spec);
```

Comment:
This change is unrelated, right? It ensures ad clicks are counted once, even though a single GET request ends up triggering the activity observer multiple times.

Reply:
Thanks, I've added that back in for now.

I also fixed the message reporting in bootstrap.js to include the message name as well as the data.